addons/website/views/res_config_settings_views.xml

```diff
@@ -370,7 +370,10 @@
                                     <div class="text-muted mt8">
                                         Sitemap.xml: Help search engine crawlers to find out what pages are present and which have recently changed, and to crawl your site accordingly. This file is automatically generated by Odoo.
                                     </div>
-                                    <div class="mt4">
+                                    <div class="alert alert-primary mt4" role="alert">
```

Comment:
That's really an unusual and ugly alert imo, let's fit the Odoo style
![image](https://github.com/odoo/odoo/assets/30048408/76a2cdce-578f-44d7-b255-3701395d1e42)

```suggestion
                                    <div class="alert alert-info mt4" role="alert">
```
![image](https://github.com/odoo/odoo/assets/30048408/31deacf5-c146-434d-9103-d48b6960e593)


There is almost 0 occurences of primary alert in Odoo
![image](https://github.com/odoo/odoo/assets/30048408/8464b87b-a86f-4d44-afec-e8632c6a1674)